packages/account/src/connectors/fuel-connector.ts

```diff
@@ -90,6 +90,7 @@ export abstract class FuelConnector extends EventEmitter implements Connector {
   metadata: ConnectorMetadata = {} as ConnectorMetadata;
   connected: boolean = false;
   installed: boolean = false;
+  external: boolean = true;
```

Comment:
Hey @arthurgeron,

Don't suppose we could get a little bit more information on what determines a connector to be external or internal?

Reply:
External connectors are third-parties that do not support the Fuel Network.
Non-Native: Wallet Connect, Solana Connector
Native: Fuelet, Burner, Fuel

Reply:
I've added the property to the docs